packages/rum/src/domain/record/viewports.spec.ts

```diff
@@ -14,11 +14,9 @@ describe('layout viewport', () => {
     window.scrollTo(0, 0)
   })
 
-  describe('get window width and height', () => {
+  describe('get window height', () => {
     it('normalized scroll matches native behaviour', () => {
-      const initialInnerWidth = getWindowWidth()
       const initialInnerHeight = getWindowHeight()
-      expect(initialInnerWidth).toBe(window.innerWidth)
       expect(initialInnerHeight).toBe(window.innerHeight)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏no need to test width anymore?

Reply:
Originally thinking it's low value being redundant and it's quite the headache to wrangle the various edge case behaviours that using the Visual Viewport API solves. innerWidth/Height should include the thickness of the sidebar while `visualViewport.width` excludes it- however conformance isn't correct with our device browsers.

But on second glance we should put in some work to describe the issues so it's been re-added with explicit handling of the scrollbar thickness by checking if there is a match with or without scrollbars included - IFF the document is scrollable.